extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -1548,34 +1415,25 @@ public function getPostedAsBot() {
 	public function setInCommentsIndex( $prop, $value, $useMaster = false ) {
 		$commentId = $this->getId();
 		if ( !empty( $commentId ) ) {
-			$commentsIndex = $this->getCommentsIndex();
-			if ( $commentsIndex instanceof CommentsIndex ) {
+			$commentsIndexEntry = $this->getCommentsIndexEntry();
+			if ( $commentsIndexEntry instanceof CommentsIndexEntry ) {
 				switch ( $prop ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)